src/tox/tox_env/python/pip/req/file.py

```diff
@@ -333,7 +333,7 @@ def _break_args_options(line: str) -> Tuple[str, str]:
         args = []
         options = tokens[:]
         for token in tokens:
-            if token.startswith("-") or token.startswith("--"):
+            if token.startswith("-"):
```

Comment:
While this won't make a difference, maybe the old version showed the intention better? hm... not sure.

Reply:
First I rewrote this to equivalent `token.startswith(("-", "--"))` but then saw its redundancy. The original intention could be written in a comment WHY we expect both `-` and `--`. Otherwise it's to be re-read to search for a typo.